packages/contracts/utils/ens.ts

```diff
@@ -34,6 +34,14 @@ export async function setupENS(domains: string[], hre: EHRE): Promise<any> {
   const resolver = await deployments.get('PublicResolver');
 
   for (let i = 0; i < domains.length; i++) {
+    console.log(`Registering subdomain ${domains[i]}`);
+
+    const resolved = await ens.resolver(ensDomainHash(domains[i]));
+    if (resolved === resolver.address) {
```

Comment:
could you call this `resolver` ? `resolved` seems like something was `resolved`.

Reply:
done

Reply:
I had to change it to `resolvedResolver` because `resolver` is used by the Resolver deployment earlier in the script